### PR TITLE
In-product onboarding

### PR DESCRIPTION
**Pull request by victorwuky**

### CODE REVIEW

File: docs/rollout/in-product-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- As the DevEx person, the system provides you real-time state of where each user is in their individual onboarding, as well as metrics of how past users performed at certain steps. You see bottlenecks and improve onboarding over time from this insight.
+
+## Future enhancement
+
+Tilt doesn't have in-product onboarding yet. If this sounds interesting to you, [schedule a chat](https://calendly.com/dbentley/tilt-enterprise) and let us know which parts of onboarding you'd want to see in Tilt.
+
+## Custom scripts
+
+In the meantime, consider using offering custom scripts to help a new user install the preferred [local Kubernetes cluster](../choosing_clusters.html) or [install](../install.html)/[upgrade](../upgrade) the preferred version of Tilt. You can also take advantage of [Tiltfiles being written in the Python-dialect of Starklark](../tiltfile_concepts) to create custom logic to help new users when the first run Tilt.
```
Comment (dbentley) [inline docs/rollout/in-product-onboarding.md:22]: I'd switch the order of Custom Scripts and Future Enhancements.

I think it's possible to do an alright job of this using local_resource, and I'd love to explore it with a team. So maybe also expand it to mention that you could do something and we'd pair with you or some such?

File: docs/rollout/in-product-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: In-product Onboarding
+layout: rollout
+---
+
+## Prefer in-product onboarding
+
+Tilt is a powerful tool, and thus requires some investment on the part of a new user to learn the basics. We recommend providing a [transition guide](../rollout/introduce-tilt) to help developers with onboarding. Ideally, many of the onboarding steps can be driven by _in-product onboarding_ instead, where Tilt itself guides the new user through initial set up, and usage of features, from basic to advanced.
```
Comment (dbentley) [inline docs/rollout/in-product-onboarding.md:8]: Love linking to the transition guide.

I think you'd still want much of what's in a transition guide even with in-product onboarding: describing why it's better, etc.

**Review by dbentley — APPROVED** [review]

File: docs/rollout/in-product-onboarding.md (hunk 1 above)
Comment (victorwuky) [reply docs/rollout/in-product-onboarding.md]: Done.

File: docs/rollout/in-product-onboarding.md (hunk 2 above)
Comment (victorwuky) [reply docs/rollout/in-product-onboarding.md]: Fixed. I made in-product onboarding complementary to the transition guide, not replacing it.

**Pull request merged**